Source/ACE.Entity/AceObject.cs

```diff
@@ -770,6 +773,13 @@ public void SetStringProperty(PropertyString propertyId, string value)
 
         public List<AceObjectPropertiesString> StringProperties { get; set; } = new List<AceObjectPropertiesString>();
 
+        public List<AceObjectPropertiesAttribute> AceObjectPropertiesAttributes { get; set; } = new List<AceObjectPropertiesAttribute>();
```

Comment:
If we're going to add these, we need the reconcile them with the CreatureAbility stuff.  Please add a TODO comment for it.